src/main/java/com/google/research/bleth/simulator/GlobalResolver.java

```diff
@@ -0,0 +1,82 @@
+package com.google.research.bleth.simulator;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.HashBiMap;
+import com.google.common.collect.ImmutableBiMap;
+import com.google.common.collect.Multimap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Tracing Simulation's Resolver, which receives information from its observers and estimate the beacons' locations according to it. */
+public final class GlobalResolver implements IGlobalResolver {
+    private EstimatedBoard estimatedBoard;
+    private Multimap<Transmission, Location> currentRoundTransmissions = ArrayListMultimap.create();
+    private Map<Transmission, Beacon> TransmissionsToBeacons;
+    private Map<Beacon, Location> beaconsToEstimatedLocations = new HashMap<>();
+
+    /**
+     * A wrapper method to create new global resolver for a tracing simulation.
+     * The new resolver has a board for storing the estimated beacons' locations.
+     * @param rowsNum is number of rows of the simulation's board.
+     * @param colsNum is number of columns of the simulation's board.
+     * @param beacons is a list of the simulation's beacons.
+     */
+    public static GlobalResolver createResolver(int rowsNum, int colsNum, List<Beacon> beacons) {
```

Comment:
Nit: since this method is inside a "resolver" class, prefer to just call it "create" (part of DRY: no need to repeat the "resolver")